advanced/events.md

```diff
@@ -352,3 +324,32 @@ The following event is provided by the [Word Count plugin]({{ site.baseurl }}/pl
 | Name            | Data                                                                                    | Description                                  |
 | --------------- | --------------------------------------------------------------------------------------- | -------------------------------------------- |
 | WordCountUpdate | `{ wordCount: { words: number, characters: number, charactersWithoutSpaces: number } }` | Fired when the editor word count is updated. |
+
+
+## Editor Manager events
+
+The following events are used for editor management. These events are used differently, as shown in [Examples: Handling Editor Manager events](#exampleshandlingeditormanagerevents).
```

Comment:
"differently to other events" maybe? might just be because I jumped straight to this section though, but my brain went "different to what?"

Reply:
fixed